openverse_catalog/dags/providers/provider_api_scripts/wikimedia_commons.py

```diff
@@ -46,6 +46,7 @@ class WikimediaCommonsDataIngester(ProviderDataIngester):
         "audio": prov.WIKIMEDIA_AUDIO_PROVIDER,
     }
     endpoint = f"https://{HOST}/w/api.php"
+    delay =0.1
```

Comment:
This change was suggested only for testing in https://github.com/WordPress/openverse-catalog/issues/727 and should not be included in this changeset:
```suggestion
```

Reply:
Hey I have removed the delay=0.1 in the latest commit

Reply:
I still see 17 files total modified by the diff, when I believe only `slack.py` should be affected by this change. Can you revert the changes that have been made to the other files? Here is a guide for [undoing things in git](https://git-scm.com/book/en/v2/Git-Basics-Undoing-Things) if you haven't encountered reverting changes before. Let us know if you need any help doing that 🙂